kernel/src/process/posix_thread/thread_local.rs

```diff
@@ -83,6 +90,88 @@ impl ThreadLocal {
     pub fn sig_stack(&self) -> &RefCell<Option<SigStack>> {
         &self.sig_stack
     }
+
+    pub fn fpu(&self) -> ThreadFpu<'_> {
+        ThreadFpu(self)
+    }
+}
+
+/// The current state of `ThreadFpu`.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```

Comment:
The doc comments should be expanded. A high-level overview of these states is lacking here.